### ISSUE TITLE
Fortran Issues when using interaction terms in Stage 2 analysis

### ISSUE THREAD

**Issue by sweeptime** [reach-lab/MixWildGUI#149]
Dear Don and Rachel,

I hope this email finds you well and you all enjoyed the beautiful summer time so far. I am writing to you because I noticed that there may be some issues in MixWILD Fortran. The issues occurred when we used interaction effects in Stage 2 analysis. Please see my attachment for more information. Thanks!

[Bug_report_Weilin_210712.pdf](https://github.com/reach-lab/MixWildGUI/files/6850064/Bug_report_Weilin_210712.pdf)
[data_and_def.zip](https://github.com/reach-lab/MixWildGUI/files/6850071/data_and_def.zip)

Best,
Wei-Lin

**Comment by hedeker** [comment]
yes, thanks for finding this.  i think that the program is not creating these three-way interaction terms correctly in creating the datasets for these stage 2 runs.  so, this needs to be remedied.
